fv3core/utils/grid.py

```diff
@@ -822,12 +853,11 @@ def from_sizer_and_communicator(
         domain = sizer.get_extent(
             [fv3gfs.util.X_DIM, fv3gfs.util.Y_DIM, fv3gfs.util.Z_DIM]
         )
-        south_edge = cube.tile.on_tile_bottom(cube.rank)
-        north_edge = cube.tile.on_tile_top(cube.rank)
-        west_edge = cube.tile.on_tile_left(cube.rank)
-        east_edge = cube.tile.on_tile_right(cube.rank)
+        south_edge = cube.tile.partitioner.on_tile_bottom(cube.rank)
+        north_edge = cube.tile.partitioner.on_tile_top(cube.rank)
+        west_edge = cube.tile.partitioner.on_tile_left(cube.rank)
+        east_edge = cube.tile.partitioner.on_tile_right(cube.rank)
```

Comment:
Is this needed? If not, can you please revert this change? When possible, you want to make calls to only attributes of objects you have access to in a function, not attributes of their attributes. That way, you only have to reason about the classes of your immediate objects when reading a code, not also their dependents. In several cases, the Communicatior/Partitioner objects are not set up to allow this (which we could perhaps fix).